docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+  
+### Constructing elements
+The signatures of the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XAttribute> constructors let you pass the contents of the element or attribute as arguments to the constructor. Because one of the constructors takes a variable number of arguments, you can pass any number of child elements. Of course, each of those child elements can contain their own child elements. For any element, you can add any number of attributes.  
+  
+When adding <xref:System.Xml.Linq.XNode> (including <xref:System.Xml.Linq.XElement>) or <xref:System.Xml.Linq.XAttribute> objects, if the new content has no parent, the objects are simply attached to the XML tree. If the new content already is parented, and is part of another XML tree, the new content is cloned, and the newly cloned content is attached to the XML tree. The last example in this topic demonstrates this.  
+  
+To create a `contacts`<xref:System.Xml.Linq.XElement>, you could use the following code:  
+  
+```csharp  
+XElement contacts =  
+    new XElement("Contacts",  
+        new XElement("Contact",  
+            new XElement("Name", "Patrick Hines"),   
+            new XElement("Phone", "206-555-0144"),  
+            new XElement("Address",  
+                new XElement("Street1", "123 Main St"),  
+                new XElement("City", "Mercer Island"),  
+                new XElement("State", "WA"),  
+                new XElement("Postal", "68042")  
+            )  
+        )  
+    );  
+```  
+  
+If indented properly, the code to construct <xref:System.Xml.Linq.XElement> objects closely resembles the structure of the underlying XML.  
+  
+### XElement constructors  
+The <xref:System.Xml.Linq.XElement> class uses the following constructors for functional construction. Note that there are some other constructors for <xref:System.Xml.Linq.XElement>, but because they are not used for functional construction they are not listed here.  
+  
+|Constructor|Description|  
+|-----------------|-----------------|  
+|`XElement(XName name, object content)`|Creates an <xref:System.Xml.Linq.XElement>. The `name` parameter specifies the name of the element; `content` specifies the content of the element.|  
+|`XElement(XName name)`|Creates an <xref:System.Xml.Linq.XElement> with its <xref:System.Xml.Linq.XName> initialized to the specified name.|  
+|`XElement(XName name, params object[] content)`|Creates an <xref:System.Xml.Linq.XElement> with its <xref:System.Xml.Linq.XName> initialized to the specified name. The attributes and/or child elements are created from the contents of the parameter list.|  
+  
+The `content` parameter is extremely flexible. It supports any type of object that is a valid child of an <xref:System.Xml.Linq.XElement>. The following rules apply to different types of objects passed in this parameter:  
+  
+- A string is added as text content.  
+  
```

Comment:
```suggestion
```

Reply:
I don't understand this change. Is it the blank line between list elements? Both the C# and the VB versions had this.

Reply:
Yes, I was just suggesting to remove.